pkg/apm/tracetracker/tracker.go

```diff
@@ -23,6 +23,14 @@ var DefaultDimsToSyncSource = map[string]string{
 	"kubernetes_pod_uid": "kubernetes_pod_uid",
 }
 
+// DefaultDimsToAddToSpans are the default dimensions to add as span tags for correlated environments
+var DefaultDimsToAddToSpans = map[string]string{
```

Comment:
Just put this in the `tracetracker` package since it isn't used at all for APM dimension update stuff.